_drafts/product-details.js

```diff
@@ -11,7 +11,7 @@ import {
     SIZE_GUIDE_ATTRIBUTES_IDS, INGREDIENTS_ATTRIBUTE_IDS,
     NUTRITION_ATTRIBUTE_IDS
 } from './attributes-mapper.js';
-import { fetchPlaceholders, sampleRUM } from '../../scripts/aem.js';
+import { fetchPlaceholders, sampleRUM } from '../../scripts/commerce.js';
```

Comment:
@bdenham I'm not clear how this is being used. Can you validate this won't break anything?